src/platform/apollolake/apollolake.x.in

```diff
@@ -406,9 +398,9 @@ SECTIONS
     *(.gnu.version)
     _text_end = ABSOLUTE(.);
     _etext = .;
-  } >sof_text :sof_text_phdr
+  } >sof_fw :sof_fw_phdr
 
-  .rodata : ALIGN(4)
+  .rodata : ALIGN(4096)
```

Comment:
What's the reasoning behind the section alignment to 4K (others also be aligned to 4k too) ?

Reply:
@lgirdwood The FW manifest assumes that its "segments" are aligned, so every section that begins that FW manifest "segment" (text/rodata/bss) is aligned. It is historically connected with the virtual mem address space management (via TLB).
Regarding SUE, yes this is a good idea that SUE follows the CNL approach. Once it is done and tested for CNL which is the first iteration goal, should be very easy to replicate by a SUE developer.